src/include/nfs/ui.rb

```diff
@@ -167,6 +166,8 @@ def GetFstabEntry(fstab_ent, existing)
         break if [:ok, :cancel].include?(ret)
       end
 
+      @hosts = form.hosts
```

Comment:
So if a result of `[]` is cached, all future uses of the "choose" button will automatically report "No NFS server has been found on your network" without really re-trying. Isn't it? If that's the case, is that really the historical behavior?

Reply:
Yes, that is the historical behavior, see https://github.com/yast/yast-nfs-client/blob/SLE-15-SP3/src/include/nfs/ui.rb#L347.